internal/backend/discussion.go

```diff
@@ -258,9 +258,95 @@ func (d *delphisBackend) UpdateDiscussion(ctx context.Context, id string, input
 
 	updateDiscussionObj(discObj, input)
```

Comment:
I have a feeling we'll want to do this flow in a tx

Reply:
unfortunately we can't blend a gorm tx with a postgres tx. gorm also automatically runs their statements in a transaction